tests/integration/conftest.py

```diff
@@ -602,3 +602,74 @@ def create_file_structure_function(get_files_list):
     yield
 
     delete_file_structure(get_files_list)
+
+
+@pytest.fixture(scope='module')
+def daemons_handler(get_configuration, request):
+    """Handler of Wazuh daemons.
+
+    It uses `daemons_handler_configuration` of each module in order to configure the behavior of the fixture.
+    The  `daemons_handler_configuration` should be a dictionary with the following keys:
+        daemons (list, optional): List with every daemon to be used by the module. In case of empty a ValueError
+            will be raised
+        wazuh_control (boolean): Configure to restart all wazuh services. Default `False`.
+        ignore_errors (boolean): Configure if errors in daemon handling should be ignored. This option is available
+        in order to use this fixture along with invalid configuration. Default `False`
+
+    Args:
+        get_configuration (fixture): Gets the current configuration of the test.
+        request (fixture): Provide information on the executing test function.
+    """
+    daemons = []
+    ignore_errors = False
+    wazuh_control = False
```

Comment:
Can you change this for `restart_all` or something similar. `wazuh_control` is a bit confusing